client/src/pages/Connect/Connect.css

```diff
@@ -0,0 +1,35 @@
+.get-started-banner {
+    width: 100vw;
+    height: 36vw;
+}
+
+#get-started-body {
+    display: flex;
```

Comment:
```suggestion
    display: flex;
    padding-top: 50px;
    padding-bottom: 200px;
```

Reply:
Some padding might be helpful for spacing, especially for the "Get Connected" button

Reply:
Looks great! Accepted the suggestion.